### PR TITLE
fix code style issues introduced in PRs #162 and #121

### PR DESCRIPTION
**Pull request by tyrasd**
see title.

### Checklist
- [x] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#code-style) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/)
- ~~I have commented my code~~
- ~~I have written javadoc (required for public methods)~~
- ~~I have added sufficient unit and API tests~~
- ~~I have made corresponding changes to the [documentation](https://github.com/GIScience/ohsome-api/tree/master/docs)~~
- ~~I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-api/blob/master/CHANGELOG.md)~~
- ~~I have adjusted the examples in the [check-ohsome-api](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api) script or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api/-/issues/new) in the corresponding repository. More Information [here](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#check-examples).~~

### CODE REVIEW

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/ParameterDescriptions.java [hunk 1]
```diff
@@ -31,8 +31,8 @@
   public static final String FORMAT =
       "Output format geojson (for /groupBy/boundary resources only), csv, or json; default: json";
   public static final String PROPERTIES =
-      "List of possible property-groups added to each OSM-element: 'tags' and/or 'metadata' " +
-      "and/or 'contributionTypes' (only for the /contributions/{geometryType} endpoints); "
+      "List of possible property-groups added to each OSM-element: 'tags' and/or 'metadata' "
+      + "and/or 'contributionTypes' (only for the /contributions/{geometryType} endpoints); "
```
Comment (FabiKo117) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/ParameterDescriptions.java:35]: My IDE moves this line and the one below one step to the right when applying auto-formatting 🤔 
Checkstyle seems to be fine with both, the non-indented version and having an indentation 🤷‍♂️

**Review by FabiKo117 — COMMENTED** [review]
just one minor comment

**Review by bonaparten — DISMISSED** [review]

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java [hunk 2]
```diff
@@ -41,8 +41,9 @@
   public DataExtractionTransformer(boolean isContributionsLatestEndpoint,
       boolean isContributionsEndpoint, ExecutionUtils exeUtils, boolean clipGeometries,
       String startTimestamp, InputProcessingUtils utils, Set<SimpleFeatureType> simpleFeatureTypes,
-      FilterExpression filter, Set<Integer> keysInt, boolean includeTags, boolean includeOSMMetadata,
-      boolean includeContributionTypes, ElementsGeometry elementsGeometry, String endTimestamp,
+      FilterExpression filter, Set<Integer> keysInt, boolean includeTags,
+      boolean includeOSMMetadata, boolean includeContributionTypes,
```
Comment (bonaparten) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java:45]: Why is includeOSMMetadata  now in the line 45? it fits in the line 44 how it was before?

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/ParameterDescriptions.java (hunk 1 above)
Comment (tyrasd) [reply src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/ParameterDescriptions.java]: ok, let's make it consistent with the lines above. :+1:

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java (hunk 2 above)
Comment (tyrasd) [reply src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java:45]: AFAICS the line would have been over 100 characters long if `boolean includeOSMMetadata,` were left on line 44: https://jenkins.ohsome.org/job/ohsome-api/job/master/509/checkstyle/fileName.-1056595858/source.f9714832-4e01-442a-a293-54fef9cd83e7/#44

**Comment by tyrasd** [comment]
I've sneaked in a few more "easily resolvable" code style fixes in 077ce0e. Now, basically the only remaining issues found by checkstyle are javadoc related (mostly missing javadocs for public methods). :dark_sunglasses:

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java (hunk 2 above)
Comment (bonaparten) [reply src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java:45]: Ok eclipse shows the line-width marker at character 101 even if it is set at 100. That is why I didn't see it.

**Review by bonaparten — APPROVED** [review]

**Pull request merged**